src/main/scala/nl.knaw.dans.dccd.convert/DccdConvertExportApp.scala

```diff
@@ -15,10 +15,559 @@
+      //TODO think about the output format of the following conditions
+      if (first.nonEmpty && last.isEmpty)
+      //TODO should we remove "+ valB"
+        range = ((first.toInt - 1) * (-1)).toString + valB + "-" + ""
+      if (first.isEmpty && last.nonEmpty)
+      //TODO should we remove "+ valB"
+        range = "" + "-" + ((last.toInt - 1) * (-1)).toString + valB
+    }
+    if (first.contains("-") && last.contains("-").equals(false)) {
+      if (first.nonEmpty && last.nonEmpty)
+        range = ((first.toInt - 1) * (-1)).toString + valB + "-" + last + valA
+      //TODO think about the output format of the following conditions
+      if (first.nonEmpty && last.isEmpty)
+      //TODO should we remove "+ valB"
+        range = ((first.toInt - 1) * (-1)).toString + valB + "-" + ""
+      if (first.isEmpty && last.nonEmpty)
+      //TODO should we remove "+ valA"
+        range = "" + "-" + last + valA
+
+    }
+    range
+
+  }
+
+  def setDcxSpatialScheme(): String = {
+    "degrees".trim
+  }
+
+  def extractDcxSpatialX(projectName: String, dir: String, userDefinedDataPath:String): String = {
+    if ((getParsedMetadata(projectName, dir, userDefinedDataPath) \\ "lat").text.nonEmpty) {
+      (getParsedMetadata(projectName, dir, userDefinedDataPath) \\ "lat").text.trim
+    }
+    else ""
+  }
+
+  def extractDcxSpatialY(projectName: String, dir: String, userDefinedDataPath:String): String = {
+    if ((getParsedMetadata(projectName, dir, userDefinedDataPath) \\ "lng").text.nonEmpty) {
+      (getParsedMetadata(projectName, dir, userDefinedDataPath) \\ "lng").text.trim
+    }
+    else ""
+  }
+
+  def extractDcIdentifierSid(projectName: String, dir: String, userDefinedDataPath:String): String = {
+    if ((getParsedMetadata(projectName, dir, userDefinedDataPath) \\ "sid").text.nonEmpty) {
+      (getParsedMetadata(projectName, dir, userDefinedDataPath) \\ "sid").text.trim
+    }
+    else ""
+  }
+
+  def extractDcIdentifierIdentifier(projectName: String, dir: String, userDefinedDataPath:String): String = {
+    if ((getParsedMetadata(projectName, dir, userDefinedDataPath) \\ "identifier").text.nonEmpty) {
+      (getParsedMetadata(projectName, dir, userDefinedDataPath) \\ "identifier").text.trim
+    }
+    else ""
+  }
+
+  def setDcxRelationQualifier(): String = {
+    " 'references' ".trim
+  }
+
+  def setDcxRelationTitle(): String = {
+    "Digital Collaboratory for Cultural Dendrochronology (DCCD)".trim
+  }
+
+  def extractDcxRelationLink(projectName: String, dir: String, userDefinedDataPath:String): String = {
+    if ((getParsedMetadata(projectName, dir, userDefinedDataPath) \\ "sid").text.nonEmpty) {
+      "https://dendro.dans.knaw.nl/dccd/project/" + (getParsedMetadata(projectName, dir, userDefinedDataPath) \\ "sid").text.trim
+    }
+    else ""
+  }
+
+  def setDcxRelationQualifier_2(): String = {
+    " 'IsFormatOf' ".trim
```

Comment:
The single quotes need to be removed, otherwise SMD won't work!
Also first character must be lowercase, which I read in the SMD code.
So "isFormatOf".